includes/AssignPost.php

```diff
@@ -98,7 +98,7 @@
 echo "<table>";
 while($row = mysqli_fetch_row($res))
 {
-	echo "<tr><td><h2>$row[0] :</h2></td><td> <input type='text' name=$row[0] autofocus ></td></tr> <br>";
+	echo "<tr><td><h2>$row[0] :</h2></td><td> <input type='text' name=$row[0] value=$row[1] autofocus ></td></tr> <br>";
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->